README.md

```diff
@@ -6,6 +6,8 @@ This build plugin is a utility for supporting Gatsby on Netlify. To support
 build caching and Gatsby functions on Netlify, you will need to install this
 plugin for your app.
 
+> **Note:** Essential Gatsby includes functionality from the [Gatsby cache build plugin](https://github.com/jlengstorf/netlify-plugin-gatsby-cache). If you already have the Gatsby Cache plugin installed, you should [remove it](https://docs.netlify.com/configure-builds/build-plugins/#remove-a-plugin) before installing this plugin.
+
```

Comment:
Wow, that is confusing. I added a reference to this, but I didn't add a remediation step for it, just noted incompatibility. I couldn't combine the removal into one step since it's not the same removal process.